layers/lldp.go

```diff
@@ -57,6 +62,21 @@ type LLDPChassisID struct {
 	ID      []byte
 }
 
+func (c *LLDPChassisID) TLVID() byte {
```

Comment:
Please add godoc comments to all exported functions.

Reply:
Also, should probably return LLDPTLVType

Reply:
Noted.